ConfigureAwaitChecker.Lib/Checker.cs

```diff
@@ -83,7 +83,7 @@ public static bool CanHaveConfigureAwait(ExpressionSyntax expression, SemanticMo
 			var type = typeInfo.ConvertedType;
 			if (type == null)
 				return false;
-			var members = type.GetMembers(ConfigureAwaitIdentifier);
+			var members = semanticModel.LookupSymbols(expression.SpanStart, type, "ConfigureAwait", true);
```

Comment:
The `ConfigureAwaitIdentifier` should be used here.

Reply:
Fixed